CONTRIBUTING.md

```diff
@@ -1,13 +1,16 @@
-# Contributing to the steering repo
+# Contributing to Steering  
+This repo and the committee itself   
 
 This repo is for the use of the steering committee to organize itself and record decisions and processes.
 
-The right mode to contribute depends on the type of change that you are proposing.
+The right mode to contribute depends on the type of change that you are proposing or engagement level you are seeking with the committee.
 
 # Non-members
 
-Submit a PR and then send email to steering@kubernetes.io.
-To communicatie sensitive items with the steering committee privately, send an email to steering-private@kubernetes.io.
+- Submit a PR and then send email to steering@kubernetes.io.
+- If you need guidance (example: SIG Chairs, funding) or have a governance discussion topic, please join us at our monthly public meeting! By joining the mailing list, you'll also get the calendar invite. Check our [README] to get the info, too.
```

Comment:
Can we add one line to direct folks to create an issue at https://github.com/kubernetes/funding for funding-related queries? That repo is not so discoverable today. :(